test/sm2_internal_test.c

```diff
@@ -251,9 +252,35 @@ static int sm2_crypt_test(void)
             "88E3C5AAFC0413229E6C9AEE2BB92CAD649FE2C035689785DA33"))
         goto done;
 
+    /* From GM/T0003.5-2012, Appendix A part.*/
```

Comment:
I am looking at the English translation of `GM/T0003.5-2012` in [GB/T 32918.5-2016](https://github.com/alipay/tls13-sm-spec/raw/master/sm-en-pdfs/sm2/GBT.32918.5-2016.SM2-en.pdf), where this seems to be reported as part of "Annex C" rather than "Appendix A", which one is correct?

Reply:
both are correct. they are the same example, which is in the Annex A at GM/T0003.5-2012, or  in the Annex C at GB/T 32918.5-2016.

Reply:
@romen

Reply:
This comment should be changed to "Annex C": in both [`GM/T0003.5-2012`](https://www.gmssl.cn/gmssl/Tool_PDF?CMD=Down&File=GMT_0003.5-2012.pdf) and [`GB/T 32918.5-2016`](https://github.com/alipay/tls13-sm-spec/raw/master/sm-en-pdfs/sm2/GBT.32918.5-2016.SM2-en.pdf) the encryption case is covered in the C Annex.